kyo-core/shared/src/main/scala/kyo/core.scala

```diff
@@ -130,35 +130,35 @@ object core:
         def accepts[T](st: State, command: Command[T]): Boolean =
             true
 
-        def done[T](st: State, v: T): Result[T] < S
+        def done[T](st: State, v: T)(using Tag[E]): Result[T] < S
```

Comment:
These changes allows cached handlers to perform new suspensions of the effect by using the provided `Tag`. It's a downside that we'll have to specify the parameter in all handlers but I think it's worth it since it enables allocationless effect handling in more scenarios.